lib/perfectqueue/backend/rdb_compat.rb

```diff
@@ -105,7 +105,10 @@ def initialize(client, config)
         @prefetch_break_types = config[:prefetch_break_types] || []
 
         @cleanup_interval = config[:cleanup_interval] || DEFAULT_DELETE_INTERVAL
-        @cleanup_interval_count = rand(@cleanup_interval)
+        # If cleanup_interval_count <= max_request_per_child / max_acquire,
```

Comment:
Looks like this condition should be `cleanup_interval >= max_request_per_child`, isn't it?